chore: ROOT-180: Stats-related endpoints in SDK

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/17844611332)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/588
<!-- Follow Merge -->

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17683079531

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17737659048

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-180'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17737659048

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17737963197

matt-bernstein commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17801381835)
> Successfully merged:  35 files changed, 909 insertions(+), 408 deletions(-)

matt-bernstein committed:
Merge branch 'develop' into 'fb-ROOT-180'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17801381835

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17833604610

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-180'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17833604610

matt-bernstein commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17834766310)
> Successfully merged: Already up to date.

nikitabelonogov commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17839700162)

nikitabelonogov committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17839700162

nikitabelonogov committed:
Merge branch 'develop' into 'fb-ROOT-180'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17839700162

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17840376126

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-180'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17840376126

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17844367726

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17844611332

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/17846878203

jombooth reviewed (commented):
LG
